modules/Dasboard/components/Navbar/Navbar.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
Do you need the React package imported?

Reply:
Bhaiya, this came with the "rafce" command.

Reply:
Removing it should be fine.